go/client/cmd_signup.go

```diff
@@ -122,6 +122,7 @@ func (s *CmdSignup) ParseArgv(ctx *cli.Context) error {
 		s.genPGP = ctx.Bool("pgp")
 		s.genPaper = true
 		s.doPrompt = false
+		s.storeSecret = true
```

Comment:
I think this is what's causing the test failure, weirdly. It used to be that signing up in batch mode failed to store secrets, so subsequent `keybase login` commands required a password. Now we're fixing that. But the kbfsdocker tests are still trying to _supply_ a password, [here](https://github.com/keybase/kbfs-server/blob/master/kbfsdocker/rekey_test.go#L231) which winds up failing [here](https://github.com/keybase/kbfs-server/blob/master/kbfsdocker/docker_utils.go#L450) (I think).